src/setup/devnet/index.js

```diff
@@ -939,6 +1044,11 @@ async function setupDevnet(config) {
   const dockerOrRemoteTasks = await devnet.getDockerOrRemoteTask()
   await dockerOrRemoteTasks.run()
 
+  if (devnet.config.devnetType === 'remote') {
```

Comment:
shouldn't we add a check here to run this only when a network is specified?

Reply:
Those tasks would run only when you provide the snapshot url, which I believe you generally won't unless running a public node.

Reply:
Yeah but everywhere we have the network check
Please let's add it here too for more clarity, consistence, and also avoid errors by someone who might specify the snapshot vars without the proper network config

Reply:
Added: [88a5694](https://github.com/maticnetwork/matic-cli/pull/157/commits/88a56943bc7a38b12a738340a89a6db70e4bd69f)